README.md

```diff
@@ -48,3 +49,71 @@ docker-compose up -d matchbox
 | AUTH_HEADER            | Optional Authorization header for requests.                                           | null                  |
 | VALIDATOR_SERVICE_URL  | Optional FHIR Validator Service url. If not provided, will skip validator test steps. | null                  |
 | MOCK_PORT              | If running in test mode, port to run mock server.                                     | null                  |
+
+## Validation Details
+
+For each of the defined transactions, the follow elements are validated using Gherkin test scenarios:
+
+- Response status code
+- Profile validation using the FHIR validator
+- Additional content validation that aligns with the transaction specifications, using the [PactumJS API Testing Tool](https://pactumjs.github.io/#/)
+- Response creates resource at specified location (if applicable for the transaction)
+- Error conditions are satisfied (if applicable for the transaction)
+
+## Transaction Details
+
+### DDCC:VS Generation Service
+
+#### Submit Health Event
+
+This transaction submits a Health Event Request consisting of a batch Bundle of [Parameters](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCGenerateHealthCertificateParameters.html) for the [Generate Health Certificate Operation](https://worldhealthorganization.github.io/ddcc/OperationDefinition-DDCC-QuestionnaireResponse-generateHealthCertificate.html). The transaction returns a Submit Health Event Response consisting of a batch-response FHIR Bundle. To learn more about this transaction, refer to the [Submit Health Event workflow in the WHO specification](https://worldhealthorganization.github.io/ddcc/transactions.html#submit-health-event).
```

Comment:
```suggestion
This transaction submits a Health Event Request consisting of a batch Bundle of [Parameters](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCGenerateHealthCertificateParameters.html) for the [Generate Health Certificate Operation](https://worldhealthorganization.github.io/ddcc/OperationDefinition-DDCC-QuestionnaireResponse-generateHealthCertificate.html). The transaction returns a Submit Health Event Response consisting of a batch-response FHIR Bundle. To learn more about this transaction, refer to the [Submit Health Event workflow in the WHO DDCC:VS Implementation Guide](https://worldhealthorganization.github.io/ddcc/transactions.html#submit-health-event).
```

Reply:
I think this is the official, more specific name of the spec, according to https://worldhealthorganization.github.io/ddcc/index.html#home